Helper/Api.php

```diff
@@ -184,7 +189,7 @@ public function tagsUseKeywords()
         return $this->getSettingsVal(self::XML_CONTENT_USE_KEYWORDS);
     }
 
-    public function tagsUseAttributes()
+    public function isProductAttributesUsedForTagsVars()
     {
         return $this->getSettingsVal(self::XML_CONTENT_USE_ATTRIBUTES);
```

Comment:
prefer to have this as a `boolval` as well.

Reply:
@brownac this is a deprecated class, we are not utilizing it in main logic